docs/usermanual.md

```diff
@@ -348,7 +348,7 @@ to the Context Broker regarding an entity called `sen1` of type `sensor`:
 ```
 
 If the API key associated to de device is `ABCDEF`, and the device ID related to `sen1` entity is `id_sen1`, this will
-generate a message in the `/ABCDEF/id_sen1/cmd` topic with the following payload:
+generate a message in the `/json/ABCDEF/id_sen1/cmd` topic with the following payload:
```

Comment:
/json -> /ul

Anyway, not sure if in this case prefix applies, as I commented at https://github.com/telefonicaid/iotagent-ul/pull/401/files#r347519318